core-scroll-header-panel.css

```diff
@@ -26,6 +30,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+:host([fullWindow]) #mainContainer {
+  position: relative;
+  width: calc(100% - {{!drawerPanel.narrow && drawerPanel.drawerWidth}});
```

Comment:
bindings wont work in `<style>` outside of native shadow dom. You'll need to use `<core-style>` for something like this.